kv_store/lww_kvs/message.proto

```diff
@@ -73,3 +73,15 @@ message Placement_Request {
   }
   repeated Tuple tuple = 1;
 }
+
+message Storage_Update {
```

Comment:
Oh, we have an EBS subfield because we can have multiple volumes mounted on the same node, right?